containers/login/index.html

```diff
@@ -0,0 +1,89 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css" />
+    <link rel="stylesheet" href="https://zingy-ui.netlify.app/main.css">
+    <link rel="stylesheet" href="../../utils.css">
+    <title>Zingy Deal | Login</title>
+</head>
+
+<body>
+    <div class="page-wrapper">
+        <nav class="navbar bg-white flex-between no-wrap pd-xsm">
+            <a class="nav-left" href="/index.html">
+                <h3 class="pd-sm secondary-color">Zingy Deal</h3>
+            </a>
+            <input class="input input-secondary border-radius-sm search" type="text" placeholder="search">
+            <div class="flex-evenly pd-sm col-gap-2 nav-right">
+                <a href="/containers/login/index.html"><button class="btn btn-secondary">Login</button></a>
+                <a href="/containers/wishlist-page/index.html">
+                    <div class="badge-container icon-badge">
+                        <button class="wishlist"><i class="fa fa-heart-o fa-2x nav-wishlist"></i></button>
+                        <span class="rounded">1</span>
+                    </div>
+                </a>
+                <a href="/containers/cart-page/index.html">
+                    <div class="badge-container icon-badge">
+                        <button><i class="fa fa-shopping-cart fa-2x cart nav-cart"></i></button>
+                        <span class="rounded">1</span>
```

Comment:
![delete this too](https://user-images.githubusercontent.com/59862355/154427981-9e920238-74a9-4b6f-88d0-f32ef5524284.jpg)
notification number should be `on` the badge, you can change it's position so that it won't look like floating !

Reply:
<img width="1422" alt="Screenshot 2022-02-18 at 1 18 45 AM" src="https://user-images.githubusercontent.com/62658404/154559686-e7cfdf2e-67d0-46ef-88c7-5567296ec416.png">

Fixed the issue